src/server/routes/api/classrooms/index.js

```diff
@@ -255,25 +286,39 @@ router.post("/:classroomId/ask", async (req, res) => {
     return res.json({ success: false, message: "You are not a member of that classroom" });
   }
 
+  const updatedClassroom = await Classrooms.findById(req.params.classroomId);
```

Comment:
What if two people add questions at the same time? the event sent with the newQuestion object could contain the wrong question.

Reply:
I have switched this to filter questions based on user.id and the initial question. 

The only issue this could have is if the user submits two of the same question at the same time. 

In that case, the socket will give the user the same `questionId` twice, with a greater number than they have locally (since they shouldn't add two of the "same" questions) on the second time, where the client should then emit the `refresh_questions` event